client/components/login/login.js

```diff
@@ -48,7 +48,9 @@ export default class Login extends React.Component {
   componentDidMount() {
     const username = getParameterByName("username");
     const token = getParameterByName("token");
-    const {loginForm} = this.props;
+    const {loginForm, setTitle, orgName, language} = this.props;
+    setTitle(`${getText(loginForm.title, language)} - ${orgName}`);
```

Comment:
Why don't we change the `setTitle` function to accept different parameters?

eg: 

```
setTitle(component, language, orgName)
```

This allows to avoid compiling the title string in many different places so if we want to change its general structure we can do it from one place.